android/sdk/src/main/java/com/taobao/weex/bridge/WXBridgeManager.java

```diff
@@ -2948,7 +2951,74 @@ public int callUpdateAttrs(String instanceId, String ref, HashMap<String, String
 
     return IWXBridge.INSTANCE_RENDERING;
   }
+  public boolean shouldReportGPULimit() {
+    IWXConfigAdapter adapter = WXSDKManager.getInstance().getWxConfigAdapter();
+    boolean report_gpu_limited_layout = false;
+    if (adapter != null) {
+      float sample_rate_of_report = Float.parseFloat(adapter
```

Comment:
Add try catch, there is no way to guarantee that the server would return `float`, maybe it returns something can't convert to `float`